pkg/terminal/command.go

```diff
@@ -250,6 +252,18 @@ Adds or removes a path subistitution rule.
 	config alias <alias>
 	
 Defines <alias> as an alias to <command> or removes an alias.`},
+		{aliases: []string{"fmt"}, cmdFn: c.fmtCommand, allowedPrefixes: scopePrefix, helpMsg: `Controls formatting of variables.
+		
+		fmt <options> print <expression>
+		fmt <options> locals
+		fmt <options> args
+		fmt <options> vars
+		
+Available options:
```

Comment:
Adding options to print is tricky because we are accepting an arbitrary expression as argument. What do you think this should look like?

Reply:
That's a good point, that does complicate things quite a bit. It would be a bit tricky and possibly hacky to try and parse something like `-format` in the expression string. Any thoughts there for a better UX?

Reply:
I don't know, the `fmt` prefix was the best idea I had. We could add a `printf` command that parses a string as the first argument, or change `print` to check if the arguments start with `"` and do something different in that case. However what would the option to disable pretty printing of time.Time look like inside a format string?

Reply:
Sorry for taking so long to get back to this. I prefer having a `printf` command as opposed to the `fmt` command.

Reply:
I was thinking that instead of having an option to enable/disable pretty printing of time.Time and math/big types, maybe we could use type casts. I.e. have `string(t)` evaluate to a string representation of `var t time.Time`.

Reply:
For `time.Time` now that we support method calls is there anything prevent the user from formatting it themselves using the provided method? https://golang.org/pkg/time/#Time.Format

Reply:
There's a lot of limitations with function call injection, I think it's still worthwhile to have it.